cos/scripts/cos.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env sh
+####################
+set -e
+
+IAM_TOKEN=$(curl -X POST "${IBMCLOUD_IAM_ENDPOINT}" \
+    -H 'Accept: application/json' \
+    -H 'Content-Type: application/x-www-form-urlencoded' \
+    --data-urlencode 'grant_type=urn:ibm:params:oauth:grant-type:apikey' \
+    --data-urlencode "apikey=${APIKEY}" | jq -r .access_token)
```

Comment:
We should update the README to indicate that jq is a requirement to run this

Reply:
handled in https://github.com/terraform-ibm-modules/terraform-ibm-static-website/pull/2/commits/a181a56efd65d0daf35c34186841de25bddae7b6